src/main/java/com/linecorp/armeria/common/http/DefaultHttpHeaders.java

```diff
@@ -122,6 +123,12 @@ public HttpHeaders status(HttpStatus status) {
     }
 
     @Override
+    public HttpHeaders endOfStream(boolean endOfStream) {
```

Comment:
Could we make this property immutable unless it has to change after instantiation?